### PR TITLE
Greytide worldwide

### PR DESCRIPTION
**Pull request by karmaisblackandbluepilled**
Changelog
:cl:
add: The M1911 to the crafting menu.
add: M1911 ammo to the hacked autolathe.
add: ShitGlock™ with a sprite by punkinguy.
add: USSR-3 Indoctrination Manual has now been made standard across all of the revolutionary leader's kits.
add: The Syndicate has decided to include guides with pretty pictures to help in making pyrotechnics in the special operations crate.
balance: Made the iShotgun cheaper for greytide power.
balance: Makes the traitor glock start with 3 ammo mags because it was useless
/:cl:

## About The Pull Request
Every single fucking department keeps getting stronger while the greytide is getting the shaft. No more, greytide world wide! Lennon the captain!
the book sprites were made by chad punkin

## Why It's Good For The Game
my dick doesn't work

### CODE REVIEW

**Comment by Punkinguy** [comment]
you should add a special item used in the making of the Glock, maybe the grip or the slide and then add it to the maint spawn pool maybe with a VERY low chance to spawn

**Comment by mkikt4743** [comment]
@Punkinguy doesn't that circumvent the whole process of ez gun

**Comment by mkikt4743** [comment]
I'd say just make it a worse version of the glock

**Comment by Timmeye** [comment]
+1 just add a modular receiver to make it harder to craft

**Comment by MrEpicSax** [comment]
+1 

_I am a greyshirt. I am going to shoot the clown and steal his shoes. I will continue to incite violence at every opportunity. Players can be sure that I will bitch and moan over OOC when security uses lethal force to put me down after I take pot shots at the HoS and call it a prank. I will craft my weapon every round I can and proceed to be a shitter until I am inevitably perma-banned for killing multiple """valids""" and disconnecting during an ahelp after calling the admin a "RTARDED NIGER"._

A man can dream can't he?

**Comment by carlopault** [comment]
sounds OP, what is the crafting formula?

**Comment by Maroonenahmoron** [comment]
-1, there's always improvised shotguns and floor tiles, having a well known traitor weapon be accessible to everyone will spin heads everywhere.

**Comment by mkikt4743** [comment]
port a low-power 9mm pistol

**Comment by mkikt4743** [comment]
browning hi-power from sip's fallout server would be good, it's a low tier pistol

**Comment by karmaisblackandbluepilled** [comment]
get me the code and i'll replace this with that

**Comment by Punkinguy** [comment]
-1 too op to have a tator item from breaking a few windows into departments and looting maint

**Comment by carlopault** [comment]
-1 what Punkinguy said

**Comment by Punkinguy** [comment]
this also adds a layer of convolution, on the matter of if someone made or bought this Glock from the uplink, causing MASSIVE confusion and well, killings of extreme prejudice

**Comment by QuarianCommando** [comment]
+1 it worked on NT Station without too much issue. Also, replace the glock with some other 9mm handgun I guess, so punkinguy can stop having an autistic fit.

**Comment by QuarianCommando** [comment]
Or better yet, dont replace the glock. Nobody ever buys a fucking glock as traitor since it's widely regarded as shit, so we may as well keep the glock crafting recipe and either improve or replace the one in the traitor uplink later.

**Comment by wheezl** [comment]
-1, buff the improvised shotgun or make it easier to build instead. greytiders have enough tools to robust anyone already

**Comment by ExpAntique** [comment]
-1 greytides can make enough weapons, giving them a straight up glock is stupid

**Comment by mkikt4743** [comment]
hang on I gotta get the code from sips and find the browning, it did like 20 damage

**Comment by QuarianCommando** [comment]
Obviously the grey tide doesn't have enough toys, since they aren't able to perform their one and only job of keeping the blue tide in check.

**Comment by mkikt4743** [comment]
also hacked lathe already has a 9mmm and .45ACp box

**Comment by wheezl** [comment]
changing the recipe to an m1911 instead of a glock doesn't solve the underlying issue. being able to make a weapon of that tier with just a modular receiver and some junk is overpowered

**Comment by MrEpicSax** [comment]
> 
> 
> changing the recipe to an m1911 instead of a glock doesn't solve the underlying issue. being able to make a weapon of that tier with just a modular receiver and some junk is overpowered

overpowered? But heaps of other shit is hella fucked and it's different for the grey tide because, what, they're gay? I didn't know we started descriminating against gays...

**Comment by karmaisblackandbluepilled** [comment]
every department has way more op shit, up to science having guns better than the nukies with the mag rifle and 6.7mm rifle

**Comment by mkikt4743** [comment]
overpowered? what about (any gun that sci has that one-hit crits or injects deathmixes)

File: code/datums/components/crafting/recipes.dm [hunk 1]
```diff
@@ -329,15 +329,26 @@
...
+	result = /obj/item/gun/ballistic/automatic/pistol/m1911
+	reqs = list(/obj/item/stack/sheet/plasteel = 5,
+				/obj/item/weaponcrafting/receiver = 1,
+				/obj/item/stack/packageWrap = 10,
+				/obj/item/stack/sheet/plastic = 10)
+	tools = list(TOOL_SCREWDRIVER, TOOL_WRENCH, TOOL_WELDER)
+	time = 200
+	category = CAT_WEAPONRY
+	subcategory = CAT_WEAPON
+
```
Comment (yoyobatty) [inline code/datums/components/crafting/recipes.dm:340]: Don't put this code here, you can easily put it under our modular crafting/recipes.dm file or atleast put a hippie edit marker

File: code/modules/research/designs/autolathe_designs.dm [hunk 2]
```diff
@@ -693,6 +693,14 @@
...
 	category = list("hacked", "Security")
 
+/datum/design/m1911ammo
+	name = "M1911 Magazine"
+	id = "1911ammo"
+	build_type = AUTOLATHE
+	materials = list(MAT_METAL = 30000, MAT_GLASS = 1000)
+	build_path = /obj/item/ammo_box/magazine/m45
+	category = list("hacked", "Security")
+
```
Comment (yoyobatty) [inline code/modules/research/designs/autolathe_designs.dm]: ditto

Comment (karmaisblackandbluepilled) [reply code/modules/research/designs/autolathe_designs.dm]: ok gonna change that after I get finished with my current shit i'm doing

**Comment by Punkinguy** [comment]
YES, SEC BALLISTICS AT LAST! +1

**Comment by karmaisblackandbluepilled** [comment]
reminder for myself that I gotta fix an error here

**Comment by Punkinguy** [comment]
you should add some rare material like titanium or what ever so you cant just mass produce mags and have backpacks full of m1911 mags, even 500 of any rare material can kind of stop it with out miners

**Comment by QuarianCommando** [comment]
Having a bag full of ammo doesn't really mean all that much when a laser just has to go into a recharger. 

Also, detectives revolver has I finite ammo and nobody complains. The empty speed loaders give you the exact same amount of materials back as it takes to make a full one.

**Comment by mkikt4743** [comment]
@QuarianCommando no they don't you retard

**Comment by Zyzarda** [comment]
-1 we have enough improvised weaponry we need some actual weapons that arent just rarely spawned OP shit.

**Comment by mkikt4743** [comment]
fuck off we have two improvised ranged weapon and both are effectively muskets, where scifags can have a gun within ten mins that's stronger than ops

**Comment by mkikt4743** [comment]
fuck off we have two improvised ranged weapon and both are effectively muskets, where scifags can have a gun within ten mins that's stronger than ops

**Comment by mkikt4743** [comment]
whoops double post

**Comment by QuarianCommando** [comment]
@mkikt4743 they do, try it. Unless it was changed, empty speed loaders should give enough metal to the autolathe to print a new one.

**Comment by mkikt4743** [comment]
it gives 10k back and each bullet gives 500 back so it's  not actually possible to print a new loader from a fresh loader

**Comment by enthethan** [comment]
"sci weapons are stronger than nuke ops" what? nukeops get like 8 different 1 shot kill weapons

**Comment by mkikt4743** [comment]
look at how much damage (and arpen so more) that a magrifle does vs ops SMG

**Comment by mkikt4743** [comment]
also for on shot kill piercing syringes do better than rocket launchers

**Comment by QuarianCommando** [comment]
@mkikt4743 
`/obj/item/ammo_box/c38
	name = "speed loader (.38)"
	desc = "Designed to quickly reload revolvers."
	icon_state = "38"
	ammo_type = /obj/item/ammo_casing/c38
	max_ammo = 6
	multiple_sprites = 1
	materials = list(MAT_METAL = 20000)`
`/datum/design/c38
	name = "Speed Loader (.38)"
	id = "c38"
	build_type = AUTOLATHE
	materials = list(MAT_METAL = 20000)
	build_path = /obj/item/ammo_box/c38
	category = list("initial", "Security")`

**Comment by mkikt4743** [comment]
do it ingame it gives 10k

**Comment by karmaisblackandbluepilled** [comment]
i added the shitglock too +1 this now or I will cry

**Comment by karmaisblackandbluepilled** [comment]
oh shit i forgot the mags fuck shit dick penis

**Comment by karmaisblackandbluepilled** [comment]
nvm fixed

**Comment by karmaisblackandbluepilled** [comment]
wait i literally didn't test this at all, i should do that

**Comment by Punkinguy** [comment]
make the shitglock part of the sec update since dwarf wanted to make "makeshift everything"

**Comment by karmaisblackandbluepilled** [comment]
no

**Comment by carlopault** [comment]
How much damage does a shitglock do per bullet?

**Comment by mkikt4743** [comment]
shilock 17

**Comment by mkikt4743** [comment]
or shlock

**Comment by Punkinguy** [comment]
im dragging my big shlock around the floor

**Comment by mkikt4743** [comment]
sorry quarian a fullly loaded speedloader gives 20k. however if you unload the bullets the bullets are 500 and the speedlaoder is 10k

**Comment by FantasticFwoosh** [comment]
Id rather them just read a book or something on the soviet revolution imported from cargo and get the crafting recipie for the tokarev - TT30 & a scrap metal tachanka facemask by binding a welding helmet and plasteel together. Even have it self consume like a wizard spell book.

There's code precedent for learnable recipies rather than straight up powercreep to the greytide in my opinion.

**Comment by karmaisblackandbluepilled** [comment]
give me the code and I can make it learnable instead of having it be for everyone

**Comment by FantasticFwoosh** [comment]
<https://discordapp.com/channels/276455180852264960/276456582500319244/694518091106877460> bro i literally told you on discord.

**Comment by karmaisblackandbluepilled** [comment]
bro i deadass can't read, sorry

**Comment by karmaisblackandbluepilled** [comment]
ok i'm gonna keep the shit glock craftable from the start but lock some other things behind books

**Comment by karmaisblackandbluepilled** [comment]
I added the communist manual thing, I'm currently wondering where to actually add it ingame.

**Comment by karmaisblackandbluepilled** [comment]
I added it to the head revs roundstart and if they get converted, gonna try and add it to maint loot and surplus crates

**Comment by karmaisblackandbluepilled** [comment]
when I say converted, I mean converted by the book into being head revs

**Comment by karmaisblackandbluepilled** [comment]
This should be done now, all that remains is to tally the votes.

**Comment by mkikt4743** [comment]
yeah

**Comment by Timmeye** [comment]
The cost to make a glock or m1911 magazine is 30000 metal. each sheet of metal equals 2000 mat in a autolathe (I believe) that would mean you would need 15 sheets of metal just for one magazine.

**Comment by karmaisblackandbluepilled** [comment]
yeah and

**Comment by yoyobatty** [comment]
I want this all tested and then I can merge it, assuming there's enough votes.

**Comment by karmaisblackandbluepilled** [comment]
works

**Comment by AnthonyJNgo** [comment]
+3, total votes is a positive +5. Merging, if it's too much we can always revert.

**Comment by wheezl** [comment]
wtf why is all this shit in one pr lol

**Comment by wheezl** [comment]
if it works I say add it and we'll work out if it's too op later

**Comment by BobJogaIsForkinBanned** [comment]
USSR books that turn you into a gun factory worker? +1

**Comment by karmaisblackandbluepilled** [comment]
bro this is already merged, why the fuck would you comment on it

**Comment by FantasticFwoosh** [comment]
Just to :muscle: 

But really where's the sequel.

**Pull request merged**